### PR TITLE
buttons added

### PR DESCRIPTION
**Pull request by swstica**

### CODE REVIEW

**Review by prankurpandeyy — APPROVED** [review]

**Comment by prankurpandeyy** [comment]
looks good to me .

**Comment by prankurpandeyy** [comment]
looks good to me .

File: buttons/button.html [hunk 1]
```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="button.css">
+    <link rel="stylesheet" href="../components.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Document</title>
```
Comment (mehulsatardekar) [inline buttons/button.html]: Change the title to Buttons.

File: buttons/button.css [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    align-items: center;
+    padding: 2rem;
+    gap: 2rem;
+    flex-wrap: wrap;
+    margin: 1rem;
+    justify-content: center;
+
+}
+
+.button{
```
Comment (Shivani-1524) [inline buttons/button.css:28]: highly recommended to add a hover effect to your button components

File: buttons/button.html [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+</head>
+<body>
+  
+    
+    <!--navbar-->
+    <div class="navbar">
+        <div class="nav-items">
+            <a class="nav-heading" href="../index.html"><img src="../images/logo.png"></a>
```
Comment (mehulsatardekar) [inline buttons/button.html:18]: Add an alt attribute on images for alternate text.

File: buttons/button.css (hunk 2 above)
Comment (Shivani-1524) [reply buttons/button.css:28]: also change the cursor to a pointer on hover to tell the user its a clickable element

**Review by mehulsatardekar — CHANGES_REQUESTED** [review]

File: buttons/button.html [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+                    <p class="component-info">Here are some examples of buttons with icons.</p>
+                </div>
+                    <div class="flex-flow-row button-examples">
+                    <div><button class="button button-icon button-primary"><i class="fa fa-shopping-cart fa-lg"></i>Cart</button></div>
+                    <div><button class="button button-icon button-secondary"><i class="fa fa-envelope fa-lg"></i>Messages</button></div>
+                    <div><button class="button button-icon button-success"><i class="fa fa-download fa-lg"></i>Download</button></div>
+                    <div><button class="button button-icon button-warning"><i class="fa fa-trash fa-lg"></i>Delete</button></div>
+
+                </div>
+                <iframe class="iframe"
```
Comment (Shivani-1524) [inline buttons/button.html:66]: Try fixing the responsiveness of your code iframe. feel free to ping me if you need help regarding this :)

**Review by mehulsatardekar — APPROVED** [review]
good to merge after the changes🚀

**Review by Shivani-1524 — CHANGES_REQUESTED** [review]

File: buttons/button.html (hunk 4 above)
Comment (swastika0015) [reply buttons/button.html:66]: Done, thanks.

File: buttons/button.css (hunk 2 above)
Comment (swstica) [reply buttons/button.css:28]: Done, thanks.

File: buttons/button.html (hunk 3 above)
Comment (swstica) [reply buttons/button.html:18]: Will do, thanks

File: buttons/button.html (hunk 1 above)
Comment (swstica) [reply buttons/button.html]: done

File: buttons/button.html (hunk 4 above)
Comment (swstica) [reply buttons/button.html:66]: Sure, thank you:)

File: buttons/button.html [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+              </iframe>
+
+            <div class="component-details">
+                <h1 class="component-heading">Floating Buttons</h1>
+                <p class="component-info"> Here some examples of floating buttons.</p>
+            </div>
+                <div class="flex-flow-row button-examples">
+                <div><button class="button-float button-plus"><i class="fa fa-plus"></i></button></div>
+                <div><button class="button-float button-plus-sm"><i class="fa fa-plus fa-2x"></i></button></div>
+                <div><button class="button-float button-warning"><i class="fa fa-trash fa-2x"></i></button></div>
```
Comment (yaswanthmyneni) [inline buttons/button.html:79]: you can give a box-shadow property for floating buttons.

**Pull request merged**